src/main/resources/application.properties

```diff
@@ -5,8 +5,18 @@ spring.jpa.hibernate.ddl-auto=create-drop
 #Comma-separated list of allowed origins
 cors.allowedOrigins=*
 
-messaging.kafka.enabled=false
-messaging.kafka.applicationId=Hello-World-Kafka-Streams
-messaging.kafka.bootstrapServers=localhost:9092
-# Comma separated list of topics
-messaging.kafka.sourceTopics=alfresco.repo.events.nodes
\ No newline at end of file
+messaging.config.enabled=false
+
+messaging.config.topic.baseUrl=http://localhost:8080
```

Comment:
Perhaps we should start to distinguish these properties from the many others that a developer would encounter when building a larger Spring Boot app and change to something like:

`alfresco.events.eventGateway.url`

then hardcode the topic endpoint as constant.  As mentioned above we'd like to move this class to an SDK and don't want to confuse developers with being able to tinker with endpoints.

The fallback properties could just be `alfresco.events.broker.*`, `alfresco.events.topic`.

Reply:
Sure, sounds good!

Reply:
Done!